scipy/stats/_continuous_distns.py

```diff
@@ -6858,9 +6858,18 @@ def _pdf(self, x, a):
     def _cdf(self, x, a):
         return x*(a + x**a)**(-1.0/a)
 
+    def _sf(self, x, a):
```

Comment:
There are some cases with large `a` and `x` close to `1` where this implementation fails but main succeeds. Try locally, e.g. `kappa3.sf(0.5, 1e5)`.

Reply:
Should we then add conditions based on the value on a?

Reply:
The answer would require more careful study of where the error is coming from. If it is unavoidable, maybe. If it can be fixed by adjusting this implementation, that would be preferable. For example, in `_isf`, it looks like `denom` overflows. In `sf`, it's `x**a`. Can those be avoided?